lib/fastlane/plugin/wpmreleasetoolkit/actions/common/circleci_trigger_job_action.rb

```diff
@@ -0,0 +1,59 @@
+module Fastlane
+    module Actions    
+        class CircleciTriggerJobAction < Action
+            def self.run(params)
+                require_relative '../../helper/ci_helper.rb'
+                
+                UI.message "Triggering job #{params[:job_params]} on branch #{params[:branch]}"
+
+                ci_helper = Fastlane::Helper::CircleCIHelper.new(
+                    params[:circle_ci_token],
+                    params[:repository],
+                    params[:organization]
+                )
+
+                res = ci_helper.trig_job(params[:branch], params[:job_params])
+                (res.code == "201") ? UI.message("Done!") : UI.user_error!("Failed to start job\nError: [#{res.code}] #{res.message}")
+            end
+  
+            #####################################################
+            # @!group Documentation
+            #####################################################
+  
+            def self.description
+                "Triggers a job on CircleCI"
+            end
+  
+            def self.available_options
+            [
+                FastlaneCore::ConfigItem.new(key: :circle_ci_token,
+                                            env_name: "FL_CCI_TOKEN",
```

Comment:
I suggest we use more explicit/descriptive env var names. It's already sometimes easy to get lost in release-toolkit when you don't have all the context, so seeing just `CCI` in a user's env vars config (e.g. `.env` files) or wherever this will be used to inject it (if users end up relying on the env var rather than passing the value directly) will end up being a bit obscure imho.

```suggestion
                                            env_name: "FL_CIRCLECI_TOKEN",
```

Of course same recommendation for all other `ConfigItems` below.